devtools/setup.cfg

```diff
@@ -1,16 +1,11 @@
 [bumpversion]
-current_version = 0.1.0
+current_version = 0.1.1
 commit = True
 tag = True
 tag_name = devtools-{new_version}
 message = [devtools] Bump version: {current_version} → {new_version}
-; for when we need to make any future dev releases
```

Comment:
Odd, i didnt delete this but it got deleted when i used bumpversion. this intended?

Reply:
Nah that's fine - we can just look thru git history :)